pkg/controller/operconfig/ingressconfig_controller.go

```diff
@@ -0,0 +1,115 @@
+package operconfig
```

Comment:
Do we even need a separate controller? Is this ever expected to change after cluster creation? Or can we just retrieve the Ingress object during the operconfig reconciliation loop and render accordingly?

Remember, we re-render every 3 minutes, so that's the longest we could go without reacting to a change.

Reply:
Users can delete and re-create the default ingress controller. I thought it was cleaner to have a separate controller than stuff all that logic into the operconfig_controller.